engines/bastion_katello/test/organizations/fenced-pages.service.test.js

```diff
@@ -6,12 +6,12 @@ describe('Factory: FencedPages', function() {
     }));
 
     it("should list all the fenced pages", function () {
-        expect(FencedPages.list().length).toBe(10);
+        expect(FencedPages.list().length).toBe(11);
```

Comment:
I'd vote for removing this test and having the addPages test check that the length is incremented by 2 but I'll let it be your call.